site/app/controllers/GlobalController.php

```diff
@@ -185,6 +185,16 @@ public function prep_course_sidebar(&$sidebar_buttons, $unread_notifications_cou
             ]);
         }
 
+        if ($this->core->getConfig()->isDebug()) {
+            $sidebar_buttons[] = new Button($this->core, [
+                "href" => $this->core->buildCourseUrl(['calendar']),
+                "title" => "Course Calendar",
+                "class" => "nav-row",
+                "id" => "nav-sidebar-calendar",
+                "icon" => "fa-calendar"
+            ]);
+        }
+
```

Comment:
I also saw this code when I was reviewing it last night and I was pretty sure that it would only be for development stuff given the isDebug() portion. I opted to leave it but if we decide on changing it I believe it's a quick fix.